tekton/release-cheat-sheet.md

```diff
@@ -19,6 +19,8 @@ the pipelines repo, a terminal window and a text editor.
     ```bash
     kustomize build tekton | kubectl --context dogfooding replace -f -
     ```
+1. Ensure the `pipeline` and `task` definitions are up-to-date in the
```

Comment:
How does a release captain do this? By verifying agains the catalog version or something else?

Reply:
Today yes 🙃 In the near future, we might want to setup some automation. @lcarva worked on an extension to renotave bot to support tekton tasks and pipeline refered by bundle. We probably could enhance this for all built-in reslovers as well.